ADD: Check for special characters in a string

binkkatal opened the pull request:
ADD: Check for special characters in a string.

The Method ContainsSpecialChars, uses regex to validate a string.
The regex being used is `^[a-zA-Z0-9]*$`

patil-ashutosh commented:
@binkkatal   Thank your for raising your pull request.

patil-ashutosh commented on regex/rstring/string.go:
```diff
@@ -5,6 +5,10 @@ import (
 )
 
+var (
+	isStringAlphabetic = regexp.MustCompile(`^[a-zA-Z0-9]*$`).MatchString
```
@binkkatal please try to address these comments.

patil-ashutosh reviewed (changes requested):
@binkkatal Please address comments. Feel free to ping if need any help

binkkatal commented:
@patil-ashutosh fixed the lint issues and squashed the commits.
ready for re review

binkkatal commented:
@patil-ashutosh  do i need to setup codecov token for this pr to be merged?

patil-ashutosh commented:
> @patil-ashutosh do i need to setup codecov token for this pr to be merged?

@binkkatal No need to setup token , I fixed CodeCov issue so no need of token.
Please refresh branch once , issue will resolve

binkkatal commented:
> > @patil-ashutosh do i need to setup codecov token for this pr to be merged?
> 
> @binkkatal No need to setup token , I fixed CodeCov issue so no need of token.
> Please refresh branch once , issue will resolve

@patil-ashutosh  i rebased with the latest master and the upload step breaks again.
can you have a look at it
